### ISSUE TITLE
Misleading error message on lb_listener

### ISSUE THREAD

**Issue by Xaving** [opentelekomcloud/terraform-provider-opentelekomcloud#2410]
Hello, 

After recently migrating the OTC provider from 1.35.5 to 1.35.8, the terraform code failed at apply, when adding listener to load balancer.

```
# module.load_balancer.opentelekomcloud_lb_listener_v3.open["mylistener"] will be created
  + resource "opentelekomcloud_lb_listener_v3" "open" {
      + admin_state_up      = true
      + advanced_forwarding = (known after apply)
      + client_timeout      = (known after apply)
      + created_at          = (known after apply)
      + default_pool_id     = (known after apply)
      + id                  = (known after apply)
      + keep_alive_timeout  = 4000
      + loadbalancer_id     = "12dsXXXXXsfsdfsdg"
      + member_timeout      = (known after apply)
      + protocol            = "TCP"
      + protocol_port       = 2225
      + security_policy_id  = (known after apply)
      + sni_match_algo      = (known after apply)
      + tls_ciphers_policy  = (known after apply)
      + updated_at          = (known after apply)
    }
```

The error message is:

```
Error: error creating LoadBalancerV3: Bad request with: [POST https://elb.eu-de.otc.t-systems.com/v3/324..2344/elb/listeners], error message: {"error_msg":"Listener v2 does not supported attribute enhance_l7policy_enable","error_code":"ELB.8902","request_id":"d278......4b82f2"}
```

The error message is quiet difficult to understand. First in the terraform configuration file we are using `opentelekomcloud_lb_loadbalancer_v3`and not an ELB. Second, we are using `lb_listener_v3` and the message mentionned a listener v2.

Affected resource:
- opentelekomcloud_lb_listener_v3

```

**Comment by anton-sidelnikov** [comment]
Hi @Xaving please attach hcl configuration. And for sure this is not imported resource? Because this error message came from API, not from terraform, maybe listener was created by another api version? Try not to set `advanced_forwarding` in config.

**Comment by Vampouille** [comment]
Hello, thanks @anton-sidelnikov for looking at this issue. Here is the code that we are trying to apply : 

```hcl
resource "opentelekomcloud_lb_listener_v3" "egress" {
  for_each = local.egress_listeners

  protocol           = "TCP"
  protocol_port      = each.value # integer
  keep_alive_timeout = 4000
  loadbalancer_id    = opentelekomcloud_lb_loadbalancer_v3.egress.id # this loadlancer was initially created with a v2 resource
  lifecycle {
    ignore_changes = [
      default_pool_id,
    ]
  }
}
```

Note that:
* The listener is new: we are trying to create a new listener, we did not import a v2 listener
* But the load balancer was migrated from a v2 ressource: create with a v2 tf resource and then imported in a v3 tf resource

Anyway an update to 1.35.16 seems to fix the issue

**Comment by Vampouille** [comment]
Maybe fixed by https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/2320/files#diff-69dfbafe80a5b0f741c3cfd2a197f524b7a9f66c4d889abaef9c2f15a60b8404L264 released in v1.35.9 ?
https://docs.otc.t-systems.com/releasenotes/terraform-provider-opentelekomcloud/#v1-35-9

**Comment by Vampouille** [comment]
@Xaving I think you can close this issue

**Comment by anton-sidelnikov** [comment]
> Maybe fixed by https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/2320/files#diff-69dfbafe80a5b0f741c3cfd2a197f524b7a9f66c4d889abaef9c2f15a60b8404L264 released in v1.35.9 ? https://docs.otc.t-systems.com/releasenotes/terraform-provider-opentelekomcloud/#v1-35-9

@Vampouille @Xaving yes, this should fix your issue, anyway good to use always latest version with latest fixes
